core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2253,6 +2253,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_READ_CACHE_TTL =
+      new Builder(Name.USER_FILE_READ_CACHE_TTL)
+          .setDefaultValue(Constants.NO_TTL)
+          .setDescription("Time to live for files loaded from UFS by a user, no ttl by default.")
+          .build();
+  public static final PropertyKey USER_FILE_READ_CACHE_TTL_EXPIRED_ACTION =
+      new Builder(Name.USER_FILE_READ_CACHE_TTL_EXPIRED_ACTION)
```

Comment:
we call it TTL_ACTION in other places, good to stay consistent

Reply:
copy that